src/python/grpcio/grpc/_cython/_cygrpc/aio/grpc_aio.pyx.pxi

```diff
@@ -13,25 +13,97 @@
 # limitations under the License.
 
 
-cdef bint _grpc_aio_initialized = 0
+cdef bint _grpc_aio_initialized = False
+# NOTE(lidiz) Theoretically, applications can run in multiple event loops as
+# long as they are in the same thread with same magic. This is not a supported
+# use case. So, the gRPC Python Async Stack should use a single event loop
+# picked by "init_grpc_aio".
+cdef object _grpc_aio_loop  # asyncio.AbstractEventLoop
+cdef int64_t _event_loop_thread_ident
+cdef str _GRPC_ASYNCIO_ENGINE = os.environ.get('GRPC_ASYNCIO_ENGINE', 'default').lower()
+grpc_aio_engine = None
+cdef object _grpc_initialization_lock = threading.Lock()
+
+
+class AsyncIOEngine(enum.Enum):
+    DEFAULT = 'default'
```

Comment:
does add any value having an enum option for `default` one when this is technically the `CUSTOM_IO_MANAGER`? maybe more the default variable as a separated variable, like:

```
_DEFAULT_IO_ENGINE = AsyncIOEngine.CUSTOM_IO_POLLER
```

Reply:
There is value for `default`, which means use the recommended mode. It gives us some freedom to change the value in future.